trunk/admin.widget.class.php

```diff
@@ -222,6 +227,24 @@ public function render_password_field()
         );
     }
 
+    public function render_template_field()
+    {
+        ?>
+        <input type="text" id="template" name="sp_settings[template]" class="regular-text"
+               value="<?php echo $this->options['template']; ?>"/><br/>
+        <small><strong>Important Notes</strong>
+            <ul>
```

Comment:
We need to figure out whether this section educates users correctly by making it clear how to use it.